src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Views/Cohort/AddDraftApprenticeship.cshtml

```diff
@@ -2,6 +2,7 @@
 
 @{
     ViewData["Title"] = "Apprenticeships";
+    ViewBag.GaData.Vpv = "/unapproved/add-apprentice";
```

Comment:
I wondered if this should be `/unapproved/add/apprentice`? (Unapproved = cohorts, so this page is there to provide apprentice details in the add cohort journey). Just a thought though, not sure what we did on the employer side. Confusingly, there are still two routes on the corresponding controller action, but the hyphenated one is wrong and will get removed soon.

Reply:
updated